views/annotations/mutation.jade

```diff
@@ -48,34 +48,63 @@ block body
 	-	}
 	br
 	div(class="container")
-		-if (error != "" || (annotation === undefined))
+		-if (error)
 			h3 Annotation not found: ##{annotation_id}
 		-else
-			h3 #{annotation.gene} mutation annotation ##{annotation.u_id}
+			h3 
+				a(href="/annotations/gene/#{annotation.gene}") #{annotation.gene}
+				|  mutation annotation ##{annotation.u_id}
 			br
 			div(class="col-lg-4 col-md-4")
+				-var measurement_type = annotation.measurement_type ? annotation.measurement_type : "--";
 				h4 Annotation Data
 					mixin tr_prop(property, value)
 						tr 
 							td #{property}
-							td !{value}
+							td 
+								block
 					table(class="table table-striped", id="annotation-table", annotation-id="#{annotation.u_id}")
 						thead
 							tr(style="background:#d3d3d3")
 								th Property
 								th Value
 						tbody
-							+tr_prop("Cancer", cancerAbbr(annotation.cancer))
-							+tr_prop("Mutation class", mutation_class(annotation.mut_class))
-							+tr_prop("Mutation type", mutation_type(annotation.mut_type))
-							+tr_prop("Protein sequence change", annotation.protein_seq_change)
-							+tr_prop("Germline?", annotation.is_germline ? annotation.is_germline : "--")
-							+tr_prop("Measurement type", annotation.measurement_type ? annotation.measurement_type : "--")
-							+tr_prop("PMID", pubmedLink(annotation.reference))
-							+tr_prop("Source", annotation.source)
+							+tr_prop("Cancer") !{cancerAbbr(annotation.cancer)}
+							+tr_prop("Mutation class") !{mutation_class(annotation.mut_class)}
+							+tr_prop("Mutation type") !{mutation_type(annotation.mut_type)}
+							+tr_prop("Protein sequence change") !{annotation.protein_seq_change}
+							+tr_prop("Germline?")
+								div(id='view-germline-div')
+									div(id='view-germline-label' style='float:left' class='entry')
+										!{annotation.is_germline}
+									span(id="edit-germline", 
+										class="glyphicon glyphicon-pencil",
+										style="float:right;cursor:pointer",
+										onclick="$(this).parent().next().show();$(this).parent().hide();")
+								div(id='edit-germline-div', style='display:none')
+									select(id="is_germline_field", class="form-control", selected="#{annotation.is_germline}")
+										option false
+										option true
```

Comment:
We need a way to set the germline field to unknown. This should be the default value.